front/app/containers/App/index.tsx

```diff
@@ -340,6 +341,7 @@ const App = ({ children }: Props) => {
               {showFrontOfficeNavbar && (
                 <ErrorBoundary>
                   <MainHeader />
+                  <MobileScrollHeader />
                 </ErrorBoundary>
```

Comment:
Since the MainHeader takes up space on the screen which other components in our platform take into account in their styling, it made more sense to me to create a separate "ScrollHeader" which is shown only on mobile and only on scroll. I felt the code was easier to understand this way rather than trying to force some additional behaviour on the existing MainHeader.

If you don't think this was the right call though, let me know :)

Reply:
Ok for sure - I'll give it a try! :) I've been trying to avoid complicating components with too many conditionals which was feedback from previous PRs, but now I've gone too far in the other direction then :laughing:

Reply:
I think you can still split out the mobile vs desktop variants, which should it make OK?

Reply:
You should indeed not create 🍝 components :D